internal/service.go

```diff
@@ -352,6 +359,20 @@ func (h *HauserService) WriteBundleToJson(bundleID int, filename string) (bytesW
 	return written, nil
 }
 
+func (h *HauserService) getValueConverter() warehouse.ValueToStringFn {
+	if h.config.StorageOnly {
+		return warehouse.ValueToString
+	}
+	return h.database.ValueToString
```

Comment:
It looks the redshift impl truncates based on `VarCharMax` in the config file and uses `time.String()` instead of `time.Format(RFC3339Micro)` for timestamps, so I don't think we can use a unified impl.

Reply:
Sorry, I meant are the implementations different between the warehouse and database for a given pairing? Are we going to allow pairing whatever two you want (even if across different providers)? I'm just wondering if `getValueConverter` should even be a single value at that point given we _always_ upload to the warehouse and would want to use its `ValueToString` there but optionally upload to a database which might have a different format, right? Although this method only allows a single value to string configuration.